db/migrate/20200226081611_add_index_to_tags_social_preview_template.rb

```diff
@@ -0,0 +1,7 @@
+class AddIndexToTagsSocialPreviewTemplate < ActiveRecord::Migration[5.2]
```

Comment:
we should probably get in the habit of using `index_exists?` for adding indexes, see https://github.com/thepracticaldev/dev.to/pull/6293

as sometimes these migrations timeout in production, we need to extra careful

Reply:
Agree, that's why I implemented that helper before, its nice bc it keeps all the if logic out of the migrations so they are still easy to read. Might want to consider reimplementing it.